site/en/blog/fedcm-chrome-116-updates/index.md

```diff
@@ -0,0 +1,519 @@
+  <figcaption>FedCM dialog rendered with "Continue to ****"</figcaption>
+</figure>
+
+{% endColumn %}
+{% Column %}
+
+<figure>
+  {% Img src="image/YLflGBAPWecgtKJLqCJHSzHqe2J2/gQpsjT9xjD8ywGQgYyqV.png", alt="FedCM dialog rendered with \"Use ****\"", width="800", height="408" %}
+  <figcaption>FedCM dialog rendered with "Use ****"</figcaption>
+</figure>
+
+{% endColumn %}
+
+{% endColumns %}
+
+Usage is simple, provide `identity.context` property one of `"signin"`
+(default), `"signup"`, `"use"` or `"continue"`.
+
+```js
+const credential = await navigator.credentials.get({
+  identity: {
+    // "signin” is the default, "signup”, "use” and "continue” 
+    // can also be used
+    context: "signup", 
+    providers: [{
+      configURL: "https://idp.example/fedcm.json",
+      clientId: "1234",
+    }],
+  }
+});
+```
+
+## IdP Sign-In Status API origin trial {: #idp-signin-status}
+
+Chrome starts an [IdP Sign-In Status API origin 
+trial](/origintrials/#/registration/-7757239251913146367) 
+on desktop from Chrome 116, followed by Android Chrome later. [Origin 
+trials](/docs/web-platform/origin-trials/) give you 
+access to a new or experimental feature, to build functionality your users can 
+try out for a limited time before the feature is made available to everyone.
+
+{% Aside 'important' %}
+
+IdP Sign-In Status API is a breaking change and will be a requirement for FedCM 
+when it's shipped.
+
+{% endAside %}
+
+[IdP Sign-In Status 
+API](https://github.com/fedidcg/FedCM/blob/main/proposals/idp-sign-in-status-api.md) 
+is a mechanism where an IdP informs the browser the user's sign-in status. With 
+this API, the browser can reduce unnecessary requests to the IdP and mitigate 
+potential timing attacks.
+
+{% Aside %}
+
+FedCM introduces a credentialed request to the [accounts list 
+endpoint](/docs/privacy-sandbox/fedcm-developer-guide/#accounts-list-endpoint). 
+This request does not send data identifying the requestor and also does not 
+allow passing through data provided by the RP, however, the browser separately 
+performs an uncredentialed request containing the RP information to the 
+`client_metadata_endpoint`, then the IdP server can correlate the uncredentialed 
+request with the credentialed request (stochastically) using timing, or other 
+fingerprinting data. See [these 
+slides](https://github.com/fedidcg/FedCM/blob/main/meetings/2022/FedCM_%20Options%20for%20the%20Timing%20Attack%20Problem%202022-08-31.pdf) 
+for more details.
+
+{% endAside %}
+
+### Inform the browser the user's sign-in status
+
+IdPs can signal the user's sign-in status to the browser by sending an HTTP 
+header or by calling a JavaScript API, when the user is signed in, or when the 
+user is signed out from all their accounts. The browser records the status as 
+one of the following: "sign-in", "sign-out" or "unknown" (default)
```

Comment:
"sign-in", "sign-out", or "unknown" (default). 

(Adding the Oxford comma and the full-stop)